### PR TITLE
Features/tag picker filtering

### PR DESCRIPTION
**Pull request by GabrielFG**
- Added the option to use a view to add your own filtering rules when fetching data.

### CODE REVIEW

File: src/Components/TagPickerGridComponent/TagPickerGridComponent/index.ts [hunk 1]
```diff
@@ -28,6 +28,8 @@ export class TagPickerGridComponent extends TagPickerBaseComponent<IInputs, IOut
         this.relationshipEntity = context.parameters.relationshipEntity.raw || "";
         this.relationshipName = context.parameters.relationshipName.raw || "";
         this.labelText = context.parameters.labelText.raw || "";
+        this.viewId = context.parameters.viewId.raw || "";
+        console.log("Filter?", context.parameters.viewId.raw);
```
Comment (JonathanThomassin) [inline src/Components/TagPickerGridComponent/TagPickerGridComponent/index.ts]: We should probably remove the logging.

File: src/Components/TagPickerGridComponent/package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
   "devDependencies": {
     "@types/react": "^16.8",
     "@types/react-dom": "^16.8",
-    "pcf-scripts": "^1",
-    "pcf-start": "^1"
+    "pcf-scripts": "~0",
+    "pcf-start": "~0"
```
Comment (JonathanThomassin) [inline src/Components/TagPickerGridComponent/package.json:23]: Normal?

File: src/Components/TreeComponent/TreeComponent/TreeBaseComponent.tsx [hunk 3]
```diff
@@ -102,7 +102,7 @@ export abstract class TreeBaseComponent<TInputs, TOutputs> implements ComponentF
         let recordData = await results[1]?.text();
 
         let entities = JSON.parse(recordData).value as ComponentFramework.WebApi.Entity[];
-
+        console.log(entities);
```
Comment (JonathanThomassin) [inline src/Components/TreeComponent/TreeComponent/TreeBaseComponent.tsx]: Remove?

File: src/Components/TagPickerGridComponent/package.json (hunk 2 above)
Comment (GabrielFG) [reply src/Components/TagPickerGridComponent/package.json:23]: Somewhat, the project wouldn't build without doing this. We'll be doing updates to dependencies after this PR anyways.

**Review by JonathanThomassin — APPROVED** [review]
lgtm

**Pull request merged**